### PR TITLE
Adds an "iris" inspired flower ticking data generator

### PR DESCRIPTION
**Pull request by dsmmcken**
Pulled the avg and stdev from the actual dataset, and then fed that into a random gauss for each species.

So conceptually you still get 3 clusters of data.

### CODE REVIEW

**Comment by dsmmcken** [comment]
actual vs randomly generated, different but broadly similar.

image

image

File: src/deephaven/plot/express/data/data_generators.py [hunk 1]
```diff
@@ -14,34 +14,136 @@
...
+    ]
+    # std [setosa, versicolor, virginica] and [sepal_length, sepal_width, petal_length, petal_width]
+    std: list[list[float]] = [
+        [0.35, 0.38, 0.17, 0.11],
+        [0.52, 0.31, 0.47, 0.20],
+        [0.64, 0.32, 0.55, 0.27],
+    ]
+
+    def random_gauss(mu: float, sigma: float) -> float:
+        return random.gauss(mu, sigma)
```
Comment (jnumainville) [inline src/deephaven/plot/express/data/data_generators.py:73]: This function doesn't seem necessary

File: src/deephaven/plot/express/data/data_generators.py [hunk 2]
```diff
@@ -14,34 +14,136 @@
...
+
+    def generate_value(seed: int, species_id: int, col: str) -> float:
+        if col_id[col] == 0:
+            random.seed(
+                5001 + seed
+            )  # set seed once per row + offset that made it look nice
+        return round(
+            random_gauss(avg[species_id][col_id[col]], std[species_id][col_id[col]]),
+            1,
+        )
```
Comment (jnumainville) [inline src/deephaven/plot/express/data/data_generators.py:83]: Reused dict access, recommend pulling out `col_id[col]` into a new variable. Maybe rename `col_id` to `col_ids` then the new variable can be `col_id`

**Review by jnumainville — CHANGES_REQUESTED** [review]

File: src/deephaven/plot/express/data/data_generators.py (hunk 1 above)
Comment (dsmmcken) [reply src/deephaven/plot/express/data/data_generators.py]: good catch, I had moved the seed out of there.

**Review by jnumainville — APPROVED** [review]

**Pull request merged**